packages/blocks/src/list-block/commands/index.ts

```diff
@@ -2,12 +2,22 @@ import type { BlockCommands } from '@blocksuite/block-std';
 
 import { convertToNumberedListCommand } from './convert-to-numbered-list.js';
 import { indentListCommand } from './indent-list.js';
+import { listToParagraphCommand } from './list-to-paragraph.js';
 import { splitListCommand } from './split-list.js';
 import { unindentListCommand } from './unindent-list.js';
 
 export const commands: BlockCommands = {
   convertToNumberedList: convertToNumberedListCommand,
+  listToParagraph: listToParagraphCommand,
   splitList: splitListCommand,
   indentList: indentListCommand,
   unindentList: unindentListCommand,
 };
+
+declare global {
+  namespace BlockSuite {
+    interface CommandContext {
+      listConvertedId?: string;
+    }
+  }
+}
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +24]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Move this declare to file `list-to-paragraph.ts` to make it more cohesive？<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7962?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
The `convertToNumberedList` command and `listToParagraph` command share the same output, what's the best practice here?

Reply:
It is difficult to decide whether to declare it repeatedly in two files or declare it once in a shared file. I might prefer the former.